src/test/footer.test.tsx

```diff
@@ -0,0 +1,9 @@
+import { render } from "@testing-library/react";
+import Footer from "../components/Footer/Footer";
+
+describe("Footer", () => {
+  it("renders Footer component", () => {
+    const { asFragment } = render(<Footer />);
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
Don't use a fragment, it's better to use a container, with fragment your snapshot contains an unnecessary wrapper `<DocumentFragment>` that doesn't exist in reality